src/nls/it/strings.js

```diff
@@ -535,15 +553,20 @@ define({
     "EXTENSION_MANAGER_REMOVE_ERROR"       : "Impossibile rimuovere una o più estensioni: {0}. {APP_NAME} si riavvierà comunque.",
     "EXTENSION_MANAGER_UPDATE"             : "Aggiorna Estensioni",
     "EXTENSION_MANAGER_UPDATE_ERROR"       : "Impossibile aggiornare una o più estensioni: {0}. {APP_NAME} si riavvierà comunque.",
+    "EXTENSION_MANAGER_DISABLE"            : "Disabilità estensione",
+    "EXTENSION_MANAGER_DISABLE_ERROR"      : "Impossibile disattivare una o più estensioni: { 0 }. { APP_NAME } sarà ricaricato nuovamente.",
```

Comment:
`Impossibile disattivare una o più estensioni: {0}. {APP_NAME} si riavvierà comunque.` (As for `EXTENSION_MANAGER_REMOVE_ERROR` and `EXTENSION_MANAGER_UPDATE_ERROR`)